backend/globaleaks/handlers/base.py

```diff
@@ -225,6 +227,24 @@ def call_handler(cls, *args, **kwargs):
 
         return wrapper
 
+    def basic_auth(self):
```

Comment:
discussed off the record; we agreed to not implement the protection given the intent of the feature that is for implementing a basic authentication just for limiting regular user access during development; clarifications about this are provided on the platform to users.